src/message/MessageContainer.js

```diff
@@ -87,21 +88,38 @@ class MessageContainer extends PureComponent {
     this.showActionSheet({ options, cancelButtonIndex: options.length - 1, callback });
   };
 
+  handleOutboxMessageLongPress = () => {
+    const { actions, message } = this.props;
+    const callback = buttonIndex => {
+      switch (buttonIndex) {
+        case 0: {
+          actions.deleteOutboxMessage(message.timestamp);
+          break;
+        }
+        default:
+          break;
+      }
+    };
+    const options = ['Delete message', 'Cancel'];
+    this.showActionSheet({ options, cancelButtonIndex: options.length - 1, callback });
```

Comment:
Most of the actions are relevant for an outbox message, why have a specific action sheet for outbox?